Formula/gr-osmosdr.rb

```diff
@@ -13,6 +13,8 @@ class GrOsmosdr < Formula
     sha256 "1316ec1150647972436f96a9d957b5c5b7889f6f962217b181e6185a939aa2e2" => :mojave
   end
 
+  disable! date "2020-01-17", because :does_not_build
```

Comment:
```suggestion
  disable! date "2020-01-17", because: :does_not_build
```
Sorry, I forgot an extra `:`